pyEDAA/UCIS/Cobertura.py

```diff
@@ -94,40 +130,49 @@ class Package:
 
 	name: str
 	classes: Dict[str, Class]
-	lines_valid: int
-	lines_covered: int
+	linesValid: int
+	linesCovered: int
 
 	def __init__(self, name: str):
 		self.name = name
 		self.classes = {}
-		self.lines_valid = 0
-		self.lines_covered = 0
+		self.linesValid = 0
+		self.linesCovered = 0
 
-	def add_statement(self, class_name: str, source_file: str, line: int, count: int) -> None:
-		try:
-			self.classes[class_name].add_statement(line, count)
-		except KeyError:
-			self.classes[class_name] = Class(source_file)
-			self.classes.get(class_name).add_statement(line, count)
+	def addClass(self, coberturaClass: Class):
+		if coberturaClass.name in self.classes:
+			raise DuplicatedClassName(f"Duplicated class name: {coberturaClass.name}")
```

Comment:
Please enclose the class name in single quotes.

```suggestion
			raise DuplicatedClassName(f"Duplicated class name: '{coberturaClass.name}'.")
```